README.md

```diff
@@ -1,133 +1,41 @@
 
-# trustourduty
-An opensource project for the Non Governmental Organisation , Organisation of charity Purposes , In which all the basic need of the NGO will be done with the help of the technology
-## Our Community
-We are grateful to the 300+ people from around the world who have contributed to the evolution of this historic document.
+# Trustourduty
 
-<a href="https://github.com/IlahiTechnologies
-/trustourduty/graphs/contributors"><img src="https://github.com/IlahiTechnologies/trustourduty/graphs/contributors.svg?width=890&button=false" /></a>
+An Open source Backend Portal for non-profit Organization to manage people and accountings https://www.trustourduty.org/
 
-https://kanbanabhinav.000webhostapp.com/
+## Why trustourduty repo is under ilahitechnologies github Organization.
 
-## Getting Started
-
-These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
-
-### Prerequisites
-
-What things you need to install the software and how to install them
-
-```
-Give examples
-```
-
-### Installing
-
-A step by step series of examples that tell you how to get a development env running
-
-Say what the step will be
-
-```
-Give the example
-```
-
-And repeat
-
-```
-until finished
-```
-
-End with an example of getting some data out of the system or using it for a little demo
-
-## Running the tests
-
-Explain how to run the automated tests for this system
-
-### Break down into end to end tests
-
-Explain what these tests test and why
-
-```
-Give an example
-```
+TrustOurDuty Organization is founded and run by people who intended to serve the society with their knowledge, Ilahi Technologies Helps and do the volunteer work to make a portal to manage TOD portal.
```

Comment:
i think it is better to mention non-profit organization everywhere. Also what is meant TOD portal